docs/iiif/index.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: default
+title: IIIF
+has_children: false
+nav_order: 5
+last_modified_date: 2022-03-14T11:40:08+0100
+---
+
+<details markdown="block">
+  <summary>
+    Inhoudstafel
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+# Over IIIF 
+
+[Image Interoperability Framework](https://iiif.io) (IIIF) is een set van open standaarden om gedigitaliseerde archieven online beschikbaar te stellen. Het zorgt voor een gebruiksvriendelijke toegang tot de objecten uit een archief.  
+Het IIIF-protocol maakt gebruik van een stabiel manifest, en voorziet een duurzame URL voor elk object in het archief. Hierdoor kunnen er later eventueel andere versies van eenzelfde object worden toegevoegd aan een collectie (bijvoorbeeld in een hogere resolutie) zonder dat de URL van dit object wijzigt. 
+
+Het IIIF-protocal bestaat uit 6 API's, die onafhankelijk van elkaar kunnen worden gebruikt. Momenteel wordt er enkel gebruik gemaakt van de Image API, wat de gebruiker in staat stelt om (delen van) afbeeldingen te bekijken, herschalen, roteren,... op een performante manier. 
+
+## Authenticatie
+
+De door meemoo voorziene IIIF-endpoints zijn publiek beschikbaar; er is dus geen authenticatie vereist. 
+
+## Endpoint
+
+Er wordt gebruik gemaakt van de de Image API versie 3.0. 
+Het meemoo Image API endpoint voldoet aan het hoogste IIIF-compliance level, nl. 2. 
+Meer info over IIIF en de Image API kan teruggevonden worden op de iiif.io [website](https://iiif.io/api/image/3.0/). 
```

Comment:
```suggestion
Meer info over IIIF en de Image API kan teruggevonden worden op de [iiif.io website](https://iiif.io/api/image/3.0/). 
```
Seems a bit odd to mention a URL as a reference to a website while it is not clickable.